src/app/utils/currentSeason.ts

```diff
@@ -0,0 +1,8 @@
+import { CURRENT_MOUNTH } from "../constants/constanst"
+
+export const currentSeason = () => {
+	if (CURRENT_MOUNTH > 2 && CURRENT_MOUNTH < 6) return "spring"
```

Comment:
Try to create service for manage weather and seasons and you can manage current season without if/else only with simple polymorphism